proto/protocolapis/astria/protocol/bridge/v1alpha1/types.proto

```diff
@@ -2,9 +2,19 @@ syntax = "proto3";
 
 package astria.protocol.bridge.v1alpha1;
 
+import "astria/primitive/v1/types.proto";
+
 // A response containing the last tx hash given some bridge address,
 // if it exists.
 message BridgeAccountLastTxHashResponse {
   uint64 height = 2;
   optional bytes tx_hash = 3;
 }
+
+message BridgeAccountInfoResponse {
```

Comment:
Small doc for what this is supposed to be the response to? I think this should mention ABCI because there is not (yet?) a matching grpc service.